src/System.Private.ServiceModel/src/Internals/System/Runtime/IOTimer.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+
+namespace System.Runtime
+{
+    public class IOTimer<TState> : IDisposable
+    {
+        private readonly Action<TState> _callback;
+        private readonly TState _state;
+        private Timer _timer;
+        private bool _enabled;
+        private DateTime _dueDateTime;
+        private readonly double MaxSkew = 100.0; // Milliseconds
```

Comment:
It was supposed to be a constant which would make the name correct. I'll fix that.